compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/AIETarget.cpp

```diff
@@ -359,6 +358,8 @@ LogicalResult AIETargetBackend::serializeExecutable(
 
     std::ifstream instrFile(static_cast<std::string>(npuInstPath));
     std::string line;
+    // Vector to store LX6 instructions.
```

Comment:
FYI: I've heard that AIE2p has LX7 instructions (not a request for change!)

Reply:
I believe it's true that forthcoming gens will have LX7 (and higher?) but the instructions themselves don't actually have anything to do with the LX arch (which is Tensilica's naming scheme for their archs and not connected to our firmware that runs _these_ instructions). But you're right the comment will be out of date.

Reply:
Lets have a clean up PR for this, I noticed you did LX[0-9] in some places, maybe we should say LX6/LX7 instead?

Reply:
I think the safest term would be `command processor` but who cares (honestly when I first started it was useful knowing that it was LX6 because there's a datasheet type thing out there for that so it clarified what people were really talking about).